internal/output/txtstyle/title.go

```diff
@@ -0,0 +1,66 @@
+package txtstyle
+
+import (
+	"unicode/utf8"
+)
+
+const (
+	// DefaultTitlePadding is the padding character length.
+	DefaultTitlePadding = 3
+)
+
+// Title represents the config of a styled title. It does not, currently,
+// support combining diactics (more info: https://play.golang.org/p/VmHyq3JJ7On).
+type Title struct {
```

Comment:
Let's have this implement output.Marshaller

http://github.com/ActiveState/cli/blob/d657478e6235fe89e8a7bc401c63e60d83349ac3/internal/output/mediator.go#L8-L8

And then only have it output data for the plain outputter, eg.

```
func (t *Title) MarshalOutput(format output.Format) interface{} {
	if format == output.PlainFormatName {
		return t.String()
	}
	return nil
}
```

Reply:
We may need to implement some type to mean "don't output this". I think `nil` will just cause it to print something "empty". Could have some type `output.Suppress` that's handled here:

http://github.com/ActiveState/cli/blob/d657478e6235fe89e8a7bc401c63e60d83349ac3/internal/output/mediator.go#L12-L12